contrib/mpi-proxy-split/Makefile_config.in

```diff
@@ -20,6 +20,23 @@ default2: default
 ## But this .o file will be linked with src/dmtcp_coordinator.o, which
 ##   is built with g++.  So, we use $(CXX) for compatibility.
 ## An alternative is to use -O2, which inlines calls to memcpy.
+mana_coordinator.o: mana_coordinator.cpp
+	${CXX} ${CXXFLAGS} `CC --cray-print-opts=cflags` -c -o $@ $<
+else ifeq ($(findstring gert,$(PLATFORM)),gert)
```

Comment:
you could try setting a variable for each of cori and gerty, like `is_cori` and `is_gert` and then doing something like `ifeq ($(or $(is_cori),$(is_gert)), 1)` to reduce the amount of duplicated code.

Reply:
Good suggestion updated the code.

Reply:
@jungan should we make `is_cori` and `is_gerty` capital for consistency as all variables in the Makefile are in caps? Otherwise, LGTM. :)

Reply:
Sure, just updated.